src/history/base.js

```diff
@@ -146,8 +146,10 @@ export class History {
             cb(err)
           })
         } else {
-          warn(false, 'uncaught error during route navigation:')
-          console.error(err)
+          if (process.env.NODE_ENV !== 'production') {
+            warn(false, 'uncaught error during route navigation:')
+            console.error(err)
```

Comment:
let's move the console.error outside of the if statement as it should be kept in production